pagopa/io-app
# fix: [IOBP-487] Transaction_operation_details swipe back handling

## Short description
Fixed a bug that would make the app not handle the swipe back correctly in `WALLET_TRANSACTION_OPERATION_DETAILS`

## List of changes proposed in this pull request
- fixed gesture handling in navigator

## How to test
In the app, activate the `beta design system` option under the `profile` section, then:
- navigate to the wallet section
- scroll down to the transaction list ("load more" in case none is rendered)
- open up a transaction's details
- click the blue cta right under the title
- after the navigation, make sure that both the swipe back in iOS and the top back button press correctly navigate to `WALLET_TRANSACTION_DETAILS`
